src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int NUMBER_OF_COLORS = 3;
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[random.nextInt(NUMBER_OF_COLORS)].name();
```

Comment:
we want to use all the colors, not the first 3
```suggestion
        return Color.values()[random.nextInt(Color.values().length)].name();
```